ipa-server/.applier/group_vars/seed-hosts.yml

```diff
@@ -0,0 +1,66 @@
+---
+namespace: "ipa"
+app_name: "ipa"
+base_openshift_url: example.com 
+realm: "EXAMPLE.COM"
+admin_password: "Password123"
+openshift_templates_version: "v1.4.5"
+ipa_templates_version: "5e55ee8a62e7c51bf8c1805b33425f48e2d8f44e"
+
+params:
+  IPA_SERVER_SERVICE: "{{ app_name }}"
+  IPA_SERVER_IMAGE: "freeipa-server:rhel-7"
+  IPA_SERVER_HOSTNAME: "{{ app_name }}.apps.{{ base_openshift_url }}"
+  IPA_SERVER_INSTALL_OPTS: "-U -r {{ realm }} --setup-dns --no-forwarders --no-ntp"
+  IPA_ADMIN_PASSWORD: "{{ admin_password }}"
+  SERVICE_ACCOUNT_USEROOT: "{{ app_name }}"
+  VOLUME_CAPACITY: 1G
+  TIMEOUT: 600
+
+openshift_cluster_content:
+- object: Environment Setup
+  content:
+  - name: Create Projects
+    template: "https://raw.githubusercontent.com/redhat-cop/openshift-templates/{{ openshift_templates_version }}/project-requests/create-project.yml"
+    action: create
+    params_from_vars:
+      NAMESPACE: "{{ namespace }}"
+      NAMESPACE_DISPLAY_NAME: "{{ namespace }}"
+    tags:
+      - ipa_project
+
+- object: Image Streams
+  content:
+  - name: "{{ app_name }}-streams"
+    file: "https://raw.githubusercontent.com/freeipa/freeipa-container/{{ ipa_templates_version }}/freeipa-server-openshift-image.json"
```

Comment:
This image stream seems to pull fedora/centos base images but up above we specify rhel-7:

```
$oc get is
NAME             DOCKER REPO                                           TAGS                                       UPDATED
freeipa-server   docker-registry.default.svc:5000/ipa/freeipa-server   centos-7,fedora-24,fedora-25 + 1 more...   2 minutes ago
```

Reply:
I guess the hash for `ipa_templates_version` doesn't include rhel. This one does though `ccf9229b9c43e9c9fa9a83ef70d1e6ade8e0f3fa`

Reply:
Actually it looks like it does, if you look at the file here: https://raw.githubusercontent.com/freeipa/freeipa-container/5e55ee8a62e7c51bf8c1805b33425f48e2d8f44e/freeipa-server-openshift-image.json , you'll see `rhel-7` at the bottom.

Reply:
Oh woops, looks like I had a local change. Not using hashes is rough.